invenio_app_ils/ui/src/semantic-ui/site/globals/site.overrides

```diff
@@ -2,17 +2,32 @@
          Site Overrides
 *******************************/
 
-.app {
-  display: flex;
```

Comment:
For the frontsite we intended to have a sticky footer, the height 100% is not going to push the content to the bottom.

Reply:
I put it back, thanks for spotting this!